script/tool/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.13.4+4
```

Comment:
Oops, I keep forgetting to rip out the metadata related to publishing this as a package. I stopped updating it once we consolidated repos.

(It's fine to leave this, but we don't actually need it or the version change any more.)